CMakeLists.txt

```diff
@@ -435,7 +435,7 @@ endif()
 #######################################################################
 # the following commands are needed to fix a problem with the libraries
 # for linux 64 bits
-if("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64")
+if("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64" OR "${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "aarch64")
   message(STATUS "x86_64 architecture detected - setting flag -fPIC")
```

Comment:
I'm sorry @jgillis, I've realised now that this message should be changed in something that states "x86_64 or ARM64 architecture detected - setting flag -fPIC" in order to not be misleading. What do you think about it?

Reply:
Sure, feel free to push another commit